lib/kino/process.ex

```diff
@@ -421,15 +430,37 @@ defmodule Kino.Process do
     {func_result, sequence_diagram}
   end
 
-  defp generate_participant_entry(pid, idx) do
+  defp get_label(pid) do
+    # :proc_lib.get_label/1 was added in OTP 27
```

Comment:
```suggestion
  @compile {:no_warn_undefined, {:proc_lib, :get_label, 1}}

  defp get_label(pid) do
    # TODO: Remove :proc_lib.get_label/1 conditional from Erlang/OTP 27+
```

Reply:
We can move the conditional to compilation time, no?

```elixir
if Code.ensure_loaded?(:proc_lib) and function_exported?(:proc_lib, :get_label, 1) do
  defp get_label(pid), do: :proc_lib.get_label(pid)
else
  defp get_label(_pid), do: :undefined
end
```

Reply:
Also works for me!

Reply:
done in https://github.com/livebook-dev/kino/pull/442/commits/07f90b1ff2b13d1d1f65b488300ef525fb37da8c